src/Contracts/TwoFactorAuthenticationProvider.php

```diff
@@ -26,7 +26,16 @@ public function qrCodeUrl($companyName, $companyEmail, $secret);
      *
      * @param  string  $secret
      * @param  string  $code
-     * @return bool
+     * @param  string  $ts
+     * @return int|bool
      */
-    public function verify($secret, $code);
+    public function verify($secret, $code, $ts = null);
+
+    /**
+     * Provide current otp code based on secret.
+     *
+     * @param  string $secret
+     * @return string
+     */
+    public function getCurrentOtp($secret);
```

Comment:
The changes to this contract are breaking changes on a stable release.

Reply:
that was only added to support tests.
I can use it without that function.